clusters/dev/flux-system/gotk-sync.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   interval: 1m0s
   ref:
-    branch: main
+    branch: chore/hos/bump
```

Comment:
```suggestion
    branch: main
```

Reply:
Yeah vurderte å reverte den, men den beviser liksom at jeg har testet dette i dev

Reply:
og hva skjer med dev når du merger og chore/hos/bump blir sletta?

Reply:
eller er det ikke automatisk sletting av branch ved merge i repoet?

Reply:
Den linjen er håndtert av bootstrap-kommandoen (se readme). Flux går inn og slenger på en commit hvis branch=/= den jeg kjører kommandoen med

Jeg kan reverte den, så slipper jeg å kjøre kommandoen nytt for dev, men kan også like godt kjøre dev og prod i samme slengen.

Reply:
kk kewl